backend/app/controllers/spree/admin/store_credits_controller.rb

```diff
@@ -78,18 +78,18 @@ def collection
         @collection = super.reverse_order
       end
 
-      def load_update_reasons
-        @update_reasons = Spree::StoreCreditUpdateReason.all.order(:name)
+      def load_reasons
+        @store_credit_reasons = Spree::StoreCreditReason.all.order(:name)
```

Comment:
maybe we should use the `active` scope here?

Reply:
done